tests/translator/test_translator.py

```diff
@@ -271,12 +287,25 @@ def test_transform_success(self, testcase, partition_with_region):
 
         print(json.dumps(output_fragment, indent=2))
 
+        # Run cfn-lint on translator test output files.
+
+        rules = cfnlint.core.get_rules([], LINT_IGNORE_WARNINGS, [])
+        output_template = cfnlint.decode.cfn_json.load(expected_filepath)
+        runner = cfnlint.Runner(rules, expected_filepath, output_template, [region])
+        matches = []
+
+        # Only run linter on normal/gov partitions. It errors on china regions
+        if testcase not in LINT_IGNORE_TESTS and partition != 'aws-cn':
+            matches = runner.run()
+        print('cfn-lint ({}): {}'.format(expected_filepath, matches))
```

Comment:
This section is really what was added, all the rest is updating output files to conform to lint rules.